functionary/ui/templates/partials/variable_row.html

```diff
@@ -10,15 +10,15 @@
         {% endif %}
     </td>
     <td>
-        {% if var_update %}
+        {% if request.session.user_can_update_variable %}
```

Comment:
Seeing this made me realize that the "Add" button in `variables_section.html` is not wrapped in this same check, so while the edit buttons for existing entries are not present, that button for creating new entries is.